src/main/java/DateTimeHandler.java

```diff
@@ -0,0 +1,31 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class DateTimeHandler {
```

Comment:
Consider writing descriptive header comments for all public classes/methods so that other developers will not be forced to read the code of the method to understand its exact behavior.